package.json

```diff
@@ -36,6 +36,7 @@
     "dk:start": "docker compose start",
     "dk:stop": "docker compose stop",
     "dk:clear": "docker compose down --volumes",
+    "dk:build": "docker build -t start-ui-web . && docker run -d --network start-ui-web_default -p 3000:3000 --name start-ui-web-container start-ui-web",
```

Comment:
The idea is to build a Docker image to deploy on some Docker running platform like k8s, we don't really want a script in `package.json` but more of a command to copy/paste in the README (just the `docker build -t start-ui-web .` part I think, and then how to use this built docker image

Reply:
tell me if in the readme.md section Dockerfile, the description and command match with your suggestion